### PR TITLE
configurable livesness and readiness probes

### PR DESCRIPTION
**Pull request by gowdab**
sometimes when resources are configured , the clamav demon takes more than 30 seconds to start up , so making it configurable through values.yml is a better option.

@desaintmartin @francoisminaud @antoinerabany

### CODE REVIEW

File: charts/clamav/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "1.8"
 description: An Open-Source antivirus engine for detecting trojans, viruses, malware & other malicious threats.
 name: clamav
-version: 1.0.11
+version: 1.0.12
```
Comment (desaintmartin) [inline charts/clamav/Chart.yaml:5]: Could you bump minor version instead of patch version? This is a new feature.

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Thanks for the contribution!

File: charts/clamav/Chart.yaml (hunk 1 above)
Comment (gowdab) [reply charts/clamav/Chart.yaml]: updated version to 1.1

**Review by desaintmartin — APPROVED** [review]

**Comment by desaintmartin** [comment]
Hm, I merged too soon, before the test passed. I'll check in a few minutes.

**Comment by gowdab** [comment]
yeah , the patch version passed without issues so hopefully this should also pass. will check too later

**Comment by gowdab** [comment]
should the tests be run again ? `✖︎ elasticsearch-logging-wiremind => (version: "1.2.2", path: "charts/elasticsearch-logging") > Error waiting for process: exit status 1`
not related to the clamav changes

**Comment by desaintmartin** [comment]
It is not related at all! I'll take care of this regression: all charts are tested for a single change in a single chart.
Thanks!

**Pull request merged**